stable/sonatype-nexus/README.md

```diff
@@ -1,14 +1,35 @@
 # Nexus
 
-[Nexus](https://www.sonatype.com/nexus-repository-oss) is the world's only repository manager with FREE support for popular formats
+[Nexus OSS](https://www.sonatype.com/nexus-repository-oss) is a free open source repository manager. It supports a wide range of package formats and it's used by hundreds of tech companies.
 
 ## Introduction
 
-This chart bootstraps a sonatype nexus instance
+This chart bootstraps a Nexus OSS deployment on a cluster using Helm.
+This setup is best configuted in [GCP](https://cloud.google.com/) since:
+- [google cloud storage](https://cloud.google.com/storage/) is used for backups
+- [GCE Ingress controller](https://github.com/kubernetes/ingress/blob/master/docs/faq/gce.md) is used for using a pre-allocated static IP in GCE.
+
+There is also the option of using a [proxy for Nexus](https://github.com/travelaudience/nexus-proxy) that authenticates Nexus against an external identity provider (only GCP IAM at the moment) which is **disabled** by default.
 
 ## Prerequisites
 
-- Kubernetes 1.6+
+- Kubernetes 1.7+ with Beta APIs enabled
```

Comment:
1.8+

We officially support the current and previous minor version of Kubernetes.